src/_guides/language/sound-faq.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: guide
+title: "Sound Dart FAQ"
+description: "Questions and answers on how and why to write sound Dart code."
+toc: false
+---
+
+This page collects some questions and answers about why and how to
+write sound Dart code. Be sure to also check out the
+[Sound Dart Guide](/guides/language/sound-dart) to learn more.
+
+### What is "strong mode" ? Is it the same as “sound Dart”?
+
+“Sound Dart”, “strong mode Dart” and “type safe Dart” are sometimes
+used interchangeably. Strong mode is Dart’s implementation of a
+sound type system. With strong mode enabled, Dart is a type safe language.
+A sound language ensures that static type annotations are actually
+correct at runtime. Strong mode is an optional mode that you enable in
+Dart Analyzer, and we encourage you to start using strong mode for your
+libraries and apps. For more information,
+see [Sound Dart Guide](/guides/language/sound-dart).
+
+### Why did the Dart team build strong mode?
+
+New tools in development for the Dart language rely on ahead-of-time (AOT)
```

Comment:
I suggest giving an example. AOT is a term that is getting overloaded recently ([see here](https://angular.io/docs/ts/latest/cookbook/aot-compiler.html)).